.github/renovate.json5

```diff
@@ -28,4 +28,13 @@
     // breaking changes
     "kill-port", // `kill-port:^2.0.0 has perf issues (#8392)
   ],
+  "customManagers": [
+    {
+      "customType": "regex",
+      "fileMatch": ["\\.[mc]?[tj]sx?$"],
```

Comment:
There is only one special case at the moment, can we match the specified file directly?

Reply:
```suggestion
      "fileMatch": ["packages\/create-vite\/src\/index\\.ts$"],
```

Reply:
Updated 👍